### ISSUE TITLE
Update collection folder (to remove/add files to DT)

### ISSUE THREAD

**Issue by criadoperez** [darktable-org/darktable#8555]
**Is your feature request related to a problem? Please describe.**
If you add a folder to DT, and then out of DT you add/remove photos, changes are not detected. This is expected behaviour, but we should be able to force a re-scan of a folder to detect changes. 
The only option at the moment is to delete the entire folder from DT and re-import them again.

Lightroom has this feature of "update folder".

**Describe the solution you'd like**
Right clicking on the collection folder and selecting "Update folder". This will do the following:
1. Delete from the catalog photos that don't exist, but running a script already available [here](https://github.com/darktable-org/darktable/blob/master/tools/purge_non_existing_images.sh), 
2. Add new photos that didn't exist int the catalog
3. Optionally, re-read xmp files and compare them with the information of the database.

**Alternatives**
It could also be a LUA script, but probably the first option is preferred.

**Comment by Nilvus** [comment]
First, it's recommanded to delete photos from darktable and not outside. Deleting from darktable will also remove it from file explorer, by using trash option and not remove one. That avoid that.

Then, it's possible to update a folder or filmroll with just a right-click and then click on "search filmroll" then search folder to update it.

I close as it is already possible. You could anwser if needed of course. Thanks.

**Comment by criadoperez** [comment]
That's great to know!!
Thank you @Nilvus .

**Comment by criadoperez** [comment]
@Nilvus , I have tested what you said and the behavior isn't as expected.
I tried both eliminating and adding new files on a folder and then clicking on "search filmroll". Deleted files aren't removed from darktable and new files are not added into darktable. 
It looks like a bug...  
Could someone else please check if it also fails for them?
I'm using latest stable 3.4.1 in Manjaro Linux.

**Comment by jcarsanfoto** [comment]
Hi,  I tried the option you say Nilvus, and it doesn't work as you say. I deleted a photo inside the closed light table the program. Once rebooted, I made the option to search for the folder to re-import, and it didn't.
On the contrary, it did work for me when I switched from distro and had to locate the images in the database. Year after year I have been locating them and everything has been restored correctly. But it doesn't work for re-importing images from a folder
I'm using latest stable 3.4.1 in Manjaro Gnome Linux.
